Source/CombatExtended/CombatExtended/Verbs/Verb_LaunchProjectileCE.cs

```diff
@@ -497,15 +497,16 @@ public virtual bool CanHitTargetFrom(IntVec3 root, LocalTargetInfo targ, out str
             if (ShooterPawn != null)
             {
                 // Check for capable of violence
-                if (ShooterPawn.story != null
-          && ShooterPawn.story.WorkTagIsDisabled(WorkTags.Violent))
+                if (ShooterPawn.story != null && ShooterPawn.story.WorkTagIsDisabled(WorkTags.Violent))
                 {
                     report = "IsIncapableOfViolenceLower".Translate(ShooterPawn.Name.ToStringShort);
                     return false;
                 }
 
                 // Check for apparel
-                if (ShooterPawn.apparel != null)
+                //pawns can use turrets while wearing shield belts, but the shield is disabled for the duration via Harmony patch (see Harmony-ShieldBelt.cs)
+                bool isTurretOperator = caster.def.building?.IsTurret ?? false;
+                if (ShooterPawn.apparel != null && !isTurretOperator)
```

Comment:
This is gonna enable turret operation regardless of any apparel, not just shield belts.

Reply:
My thinking was that mod-added apparel would implement their own Harmony postfixes in their CE patch, but now that you mention it, I can't really make that assumption: some apparel might simply be intended to not allow firing at all, rather than having something that can be toggled off to allow it.